select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2ResourcesBehavior.java

```diff
@@ -60,20 +56,5 @@ public void renderHead(Component component, IHeaderResponse response)
 		{
 			response.render(CssHeaderItem.forReference(settings.getCssReference()));
 		}
-		//i18n
-		try
-		{
-			String name = String.format("res/js/i18n/%s.js", Session.get().getLocale().toLanguageTag());
-			URL lang = getClass().getResource(name);
-			if (lang != null)
```

Comment:
Yes, but do we really need such a ckeck?

1. It adds overhead to the (guessed) more than ~90% where the configured language is correct (and existing)
2. There is a fallback to "en" anyway
3. You'll see a 404 request if you configured a non-existing language
4. User with a browser that uses an unsopported language will *always* see a warning in the logfile ...